perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_eduPersonScopedAffiliations.java

```diff
@@ -1,15 +1,40 @@
 package cz.metacentrum.perun.core.impl.modules.attributes;
 
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.User;
+import cz.metacentrum.perun.core.api.exceptions.AttributeNotExistsException;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
 import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributeCollectedFromUserExtSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.regex.Pattern;
 
 /**
  * All affiliations collected from UserExtSources attributes.
  *
  * @author Martin Kuba makub@ics.muni.cz
+ * @author Dominik Frantisek Bucik <bucik@ics.muni.cz>
  */
 @SuppressWarnings("unused")
 public class urn_perun_user_attribute_def_virt_eduPersonScopedAffiliations extends UserVirtualAttributeCollectedFromUserExtSource {
 
+	private final Logger log = LoggerFactory.getLogger(this.getClass());
+
+	//eduPersonScopedAffiliationsManuallyAssigned
+	private final String SOURCE_ATTRIBUTE_FRIENDLY_NAME_EPSA_MA = "eduPersonScopedAffiliationsManuallyAssigned";
+	private final String SOURCE_ATTRIBUTE_FRIENDLY_NAME_EPSA_MA_URN = "urn:perun:user:attribute-def:def:eduPersonScopedAffiliationsManuallyAssigned";
```

Comment:
You can use
`AttributesManager.NS_USER_ATTR_DEF + ":" + SOURCE_ATTRIBUTE_FRIENDLY_NAME_EPSA_MA `